docs/quickstarts/tpa-scanning-an-sbom-file/tpa-scanning-an-sbom-file.yml

```diff
@@ -0,0 +1,33 @@
+apiVersion: console.openshift.io/v1
+kind: QuickStarts
+metadata:
+  name: tpa-scanning-an-sbom-file
+  generation: 1
+  instructional: true
+spec:
+  displayName: Scan a software bill of materials file
+  durationMinutes: 5
+  icon: >-
```

Comment:
We are not displaying icons in the frontend. You can replace the string with `~`

Reply:
Hmm... I'm not sure I understand why.  When I look that the tiles (cards) on the [Learning Resources page](https://console.stage.redhat.com/application-services/learning-resources), the tiles there have icons.  Looking at the "openshift-ai-rhods-fasttrack" quick start, and it has a base64 "icon" string, and the icon displays on the tile.  I would like to keep the icon.